### ISSUE TITLE
file location not open. minor bug

### ISSUE THREAD

**Issue by bewam** [mrohland/plantuml#10]
Hi,

There is a syntax in plantuml that allows to select the file destination within the file. I don't have the documentation for it but if we use `@startuml{images/diagram.png}`, the result goes in the directory named images with the name diagram.png (tested on unix). And it's correctly generated.
The atom script script fails with: 'Please make sure PlantUml can write to location of original file.'
The script is looking for an evaluated path from the original filename where it shouldn't.

**Comment by mrohland** [comment]
Yes, thats how it can display the file. I'm not sure ifI  like the idea to examine the plantuml file content to extract the right filename. It would couple the plugin more tightly to the syntax of plantuml and future developments of it.

**Comment by bewam** [comment]
Maybe your package can add arguments to pass to plantuml.jar. In a similar case, it would be the -output argument to send files to a specific directory.

**Comment by mrohland** [comment]
For me the main usecase is to edit and generate the result directly out of atom. Having a config var to override would override the path for all files, however this is probably only useful for a little subset of all files which one edits throuout the day. 

Maybe a second shortcut to generate into tmp for preview could help here for more flexibility. 

Sadly I don't have any time right now to take care of this. However if you want, just open a PR and I'm happy to merge it.

**Comment by bewam** [comment]
done.
The actual version of plantuml is already capable of emitting .png file from a plantuml one. The author of plantuml has thrown a beta version to add a directory as destination and a multiple destinations support. So I implemented it this way.
Will be possible :
- `@startuml{directory/}`
- `@startuml{directory/file.png}`
- `@startuml`

In any case, your main usecase is respected.
Code may change due to my lack of acknowledgment in coffeeScript.
